R/reprex_inform.R

```diff
@@ -0,0 +1,22 @@
+reprex_quiet <- function() {
+  as.logical(Sys.getenv("REPREX_QUIET", unset = "NA"))
+}
+
+local_reprex_quiet <- function(reprex_quiet, env = parent.frame()) {
```

Comment:
I deliberately left out a default for `reprex_quiet` so that you always have to specify this. I think it makes any usage -- really just in tests, I suppose -- more self-explanatory.

Reply:
Since you only ever seem to use `local_reprex_quiet()` and `local_reprex_quiet(FALSE)` feels like a double negative, maybe it should be `local_reprex_loud()` or similar?